pkg/network/setup/podnic.go

```diff
@@ -126,7 +128,17 @@ func newPodNIC(vmi *v1.VirtualMachineInstance, network *v1.Network, handler netd
 }
 
 func (l *podNIC) setPodInterfaceCache() error {
-	ifCache := &cache.PodIfaceCacheData{Iface: l.vmiSpecIface}
+	ifCache, err := cache.ReadPodInterfaceCache(l.cacheCreator, string(l.vmi.UID), l.vmiSpecNetwork.Name)
```

Comment:
Note that `ReadPodInterfaceCache` gets the network name instead of the pod iface name, I think its belongs to the next commit.